transport/src/main/java/io/camunda/zeebe/transport/stream/impl/messages/StreamTopics.java

```diff
@@ -11,7 +11,8 @@ public enum StreamTopics {
   ADD("stream-add"),
   PUSH("stream-push"),
   REMOVE("stream-remove"),
-  REMOVE_ALL("stream-remove-all");
+  REMOVE_ALL("stream-remove-all"),
+  RESTART_STREAMS("restart-streams");
```

Comment:
:wrench: Let's keep the `stream-` prefix, so e.g. `stream-restart`. Although I would suggest `recreate` as a more accurate description, or `reregister` (but that's kind of a mouthful :sweat_smile:)